src/appleseed-max-impl/appleseedobjpropsmod/appleseedobjpropsmod.cpp

```diff
@@ -137,6 +138,10 @@ namespace
             p_default, TRUE,
             p_ui, ParamMapIdVisibility, TYPE_SINGLECHECKBOX, IDC_BUTTON_VISIBILITY_SSS,
         p_end,
+        ParamIdGenerateAssembly, L"generate_assembly", TYPE_BOOL, 0, IDS_GENERATE_ASSEMBLY,
+            p_default, TRUE,
```

Comment:
Does that mean that by default, as soon as the modifier is applied to an object, an assembly will be generated for it? I think that by default this should be false?